introduction.md

```diff
@@ -0,0 +1,4 @@
+Intor asdjh kasdhjkasdh'kj hasgk dhaskj hdakjs hkajsdh kjasgh' kjasdh' jahsd' kjhasgkjd hakjsdh' jskdkjsjdjs jkdjasjk jkaj djkasd a.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.